src/domain/valueObject/sslId.go

```diff
@@ -51,33 +43,32 @@ func NewSslIdFromSslPairContent(
 	sslCertificate SslCertificateContent,
 	sslChainCertificates []SslCertificateContent,
 	sslPrivateKey SslPrivateKey,
-) (SslId, error) {
-	var sslChainCertificatesMerged string
+) (sslId SslId, err error) {
+	sslChainCertificatesMerged := ""
 	for _, sslChainCertificate := range sslChainCertificates {
 		sslChainCertificatesMerged += sslChainCertificate.String() + "\n"
 	}
-
 	contentToEncode := sslCertificate.String() + "\n" + sslChainCertificatesMerged + "\n" + sslPrivateKey.String()
 
-	sslId, err := sslIdFactory(contentToEncode)
+	sslId, err = sslIdFactory(contentToEncode)
```

Comment:
Ok, it's the ssl content? Then value there should be called certContent or something like that. It may also be a string pointer due to its size.

Reply:
Done!
Now it's called `sslCertContent` and is a string pointer.